docs/topics/OAuth2.md

```diff
@@ -175,7 +175,7 @@ On redirect, your redirect URI will contain additional **URI fragments**: `acces
 ###### Redirect URL Example
 
 ```
-https://findingfakeurlsisprettyhard.tv/#access_token=RTfP0OK99U3kbRtHOoKLmJbOn45PjL&token_type=Bearer&expires_in=604800&scope=identify&state=15773059ghq9183habn
+https://Randomurl.com/#access_token=RTfP0OK99U3kbRtHOoKLmJbOn45PjL&token_type=Bearer&expires_in=604800&scope=identify&state=15773059ghq9183habn
```

Comment:
This site you changed it to is a real site (also, not to mention one of those "get this browser extension to verify your identity"). Same as above.